topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -542,10 +542,18 @@ If you have followed all of the steps of this tutorial, you will have two useabl
 
 {% snippet faqs/galaxy/collections_build_list_paired.md %}
 
-You can now run the **TB Variant Analysis**, launching it directly from WorkflowHub.eu on several of the public Galaxy servers.
+The workflow that we are going to use is published on [WorkflowHub.eu](https://workflowhub.eu/workflows/1035) and you can run it directly on Galaxy servers using the link below:
 
 {% snippet faqs/galaxy/workflows_run_wfh.md title="TB Variant Analysis v1.0" wfhub_id="1035" %}
 
+If the text above doesn't show a text to run or if you are not on one of the public "usegalaxy.*" servers that integrate with WorkflowHub.eu:
```

Comment:
i mean that's the whole point of the snippet, you don't need to be on one of the dot star instances. our redirection tool supports any galaxy. just click other

![Screenshot_20240611-095740](https://github.com/galaxyproject/training-material/assets/458683/0c35b346-0470-41c7-829d-4dc45b851f71)

Reply:
how can we change that page to be clearer about that possibility?

Reply:
edit: just due to the tutorial mode interface, got it. that makes sense. will look into